### ISSUE TITLE
Fails to install alongside with ESLint 0.16.x

### ISSUE THREAD

**Issue by koistya** [MoOx/eslint-loader#11]
```
npm ERR! node v0.12.0
npm ERR! npm  v2.7.0
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package eslint does not satisfy its siblings' peerDependencies req.
npm ERR! peerinvalid Peer eslint-loader@0.4.0 wants eslint@^0.15.0
```

**Comment by jdlehman** [comment]
This should be fixed with PR #12

**Comment by MoOx** [comment]
Closed by 9d52d09a31002f3197cb815813419caa552a30e4
